custom_domains/getanid/terraform.tf

```diff
@@ -0,0 +1,29 @@
+terraform {
+
+  required_version = "~> 1.2"
+  required_providers {
+    azurerm = {
+      source  = "hashicorp/azurerm"
+      version = "3.22.0"
+    }
+  }
+  backend "azurerm" {
+  }
+}
+
+provider "azurerm" {
+  features {}
+
+  skip_provider_registration = true
+  #   subscription_id            = try(local.azure_credentials.subscriptionId, null)
+  #   client_id                  = try(local.azure_credentials.clientId, null)
+  #   client_secret              = try(local.azure_credentials.clientSecret, null)
+  #   tenant_id                  = try(local.azure_credentials.tenantId, null)
+}
+
+provider "azurerm" {
+  features {}
+  version         = "3.22.0"
+  alias           = "transversal"
```

Comment:
This subscription is not actually transversal. Maybe "application" or "environment" would be a more explicit name?